### DISCUSSION TITLE
Finalize an Auth Request and get the callback URL API

### DISCUSSION THREAD

**Discussion by santhosh-stitch** [zitadel/zitadel#6408] [Q&A]
Hi,

please help us on the following issue we are using zitadel Finalize an Auth Request and get the callback URL API of the api call as follows,

api - /v2alpha/oidc/auth_requests/:authRequestId
method - post
payload - {
    "session": {
        "sessionId": "sessionId from create session",
        "sessionToken": "sessionToken from create session"
    }
}
response: {
    "code": 5,
    "message": "Not Found"
}

According to Zitadel documentation, the proper authRequestId is supplied in the aforementioned api. however, I receive the response mentioned above.
Could you please explain why this response will come back? Additionally, the aforementioned api details' functional order.

**Comment by hifabienne** [comment]
Hei @santhosh-stitch 
Did you also try the get auth request method? Just to make sure that it really exists?
https://zitadel.com/docs/apis/resources/oidc_service/oidc-service-get-auth-request
